php/class-wp-seo-social-wp-seo-filters.php

```diff
@@ -53,6 +55,84 @@ public function filter_wp_seo_options_page_menu_title() {
 	 * @param array $array Option values with default keys and values.
 	 * @return array Option values with default keys and values.
 	 */
+	public function filter_wp_seo_whitelisted_settings( $array ) {
+		$extra_fields = array(
+			'og_title',
+			'og_description',
+			'og_image',
+			'og_type',
+		);
+		return array_merge( $array, $extra_fields );
+	}
+
+	/**
+	 * Fields for integer sanitization.
+	 *
+	 * @return array Fields for sanitization.
+	 */
+	public function filter_wp_seo_sanitize_as_integer_field() {
+		$sanitize_as_integer = array(
+			'home_og_image',
+		);
+		foreach ( WP_SEO_Settings()->single_post_types as $type ) {
+			$sanitize_as_integer[] = "single_{$type->name}_og_image";
+		}
+		// Post type, taxonomy, and other archives.
+		foreach ( array_merge( WP_SEO_Settings()->archived_post_types, WP_SEO_Settings()->taxonomies ) as $type ) {
```

Comment:
Can you be 100% certain that both of those functions will return arrays?  If not you might need to type cast to use `array_merge` without a notice.